libs/net/kafka/SignalR/KafkaHubLifetimeManager`.cs

```diff
@@ -0,0 +1,495 @@
+using System.Text.Json;
+using Confluent.Kafka;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.AspNetCore.SignalR.Protocol;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using TNO.Core.Extensions;
+using TNO.Kafka.Serializers;
+
+namespace TNO.Kafka.SignalR;
+
+/// <summary>
+/// KafkaHubLifetimeManager class, provides a Kafka back-plane to support clusters.
+/// All messages are routed to Kafka.
+/// This hub is a Kafka consumer and will action the message once it consumes it.
+/// </summary>
+/// <typeparam name="THub"></typeparam>
+public class KafkaHubLifetimeManager<THub> : DefaultHubLifetimeManager<THub>, IDisposable
```

Comment:
This is where all the magic occurs to support a Kafka backpane.  It's a simple implementation compared to other examples.

Here is an example of a far more complex implementation - https://github.com/jbendson/SignalR.Kafka
I'm not certain if my implementation (especially the serializer/deserializer) will work in all cases.